server/webhook.go

```diff
@@ -242,46 +247,25 @@ func (s *Server) createSchemaUpdateIssue(ctx context.Context, repository *api.Re
 		filteredDatabaseList = databaseList
 	}
 
-	var pipelineApprovalByEnv = map[int]api.PipelineApprovalValue{}
-	{
-		// It could happen that for a particular environment a project contain 2 database with the same name.
-		// We will emit warning in this case.
-		var databaseListByEnv = map[int][]*api.Database{}
-		for _, database := range filteredDatabaseList {
-			list, ok := databaseListByEnv[database.Instance.EnvironmentID]
-			if ok {
-				databaseListByEnv[database.Instance.EnvironmentID] = append(list, database)
-			} else {
-				list := make([]*api.Database, 0)
-				databaseListByEnv[database.Instance.EnvironmentID] = append(list, database)
-			}
-
-			// Load pipeline approval policy per environment.
-			if _, ok := pipelineApprovalByEnv[database.Instance.EnvironmentID]; !ok {
-				p, err := s.PolicyService.GetPipelineApprovalPolicy(ctx, database.Instance.EnvironmentID)
-				if err != nil {
-					return nil, fmt.Errorf("failed to find pipeline approval policy for environment %v", database.Instance.EnvironmentID)
-				}
-				pipelineApprovalByEnv[database.Instance.EnvironmentID] = p.Value
-			}
-		}
-
-		var multipleDatabaseForSameEnv = false
-		for environmentID, databaseList := range databaseListByEnv {
-			if len(databaseList) > 1 {
-				multipleDatabaseForSameEnv = true
-
-				s.l.Warn(fmt.Sprintf("Ignored committed file, multiple ambiguous databases named %q for environment %d.", mi.Database, environmentID),
-					zap.Int("project_id", repository.ProjectID),
-					zap.String("file", added),
-				)
-			}
-		}
-
-		if multipleDatabaseForSameEnv {
-			return nil, nil
+	// It could happen that for a particular environment a project contain 2 database with the same name.
+	// We will emit warning in this case.
+	var databaseListByEnv = map[int][]*api.Database{}
+	for _, database := range filteredDatabaseList {
+		databaseListByEnv[database.Instance.EnvironmentID] = append(databaseListByEnv[database.Instance.EnvironmentID], database)
+	}
+	multipleDatabaseForSameEnv := false
+	for environmentID, databaseList := range databaseListByEnv {
+		if len(databaseList) > 1 {
+			multipleDatabaseForSameEnv = true
+			s.l.Warn(fmt.Sprintf("Ignored committed file, multiple ambiguous databases named %q for environment %d.", mi.Database, environmentID),
+				zap.Int("project_id", repository.ProjectID),
+				zap.String("file", added),
+			)
 		}
 	}
+	if multipleDatabaseForSameEnv {
```

Comment:
If we return both nil, then the caller will not call createIgnoredFileActivity(err) ?